### PR TITLE
Fixed: auto formatting option value in schema app dropdown

### LINKED ISSUE #1343: Schema App: Dropdown field type does not allow for spaces in value
When the user selects the Dropdown field type in schema and they are configuring the values for each dropdown option, the UI does not allow the user to have spaces or capital letters in the value field. 

The API does allow for capital letters and spaces and so the UI should also allow for the same formatting options. 

Screen Shot 2022-09-08 at 2 35 56 PM

### PR DESCRIPTION
**Pull request by JacobDiazCruz**
Closes #1343 

I believe the Option value should also be escaped in the backend to ensure prevention of XSS.

### CODE REVIEW

File: src/utility/validInput.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const validInput = (val) => {
+  return /^[a-zA-Z0-9!@#?$%\^&\s*)(+=._-]*$/.test(val);
```
[the remark this answers is not in the document]
Comment (shrunyan) [reply src/utility/validInput.js:2]: Why is this necessary? I thought the intent was to allow any value. How did we determine what "valid" input is?

[the remark this answers is not in the document]
Comment (JacobDiazCruz) [reply src/utility/validInput.js:2]: Based on my notes on last week's meeting discussion, we're going to invalidate any html special characters? But if that's not the case then I should just remove the regex validation and allow any value.

[the remark this answers is not in the document]
Comment (JacobDiazCruz) [reply src/utility/validInput.js]: Btw, you may review my latest commit:
[16bbd67](https://github.com/zesty-io/manager-ui/pull/1372/commits/16bbd671362ac5a2b7411c0d912d5db4e167d909)

**Comment by shrunyan** [comment]
closed in favor of https://github.com/zesty-io/manager-ui/pull/1866

**Pull request closed without merging**